### PR TITLE
Chris add GitHub rules

### PR DESCRIPTION
**Pull request by chrispsommers**
Signed-off-by: Chris Sommers <chris.sommers@keysight.com>

Adapted from https://github.com/Azure/DASH/blob/main/doc-github-rules.md which I co-authored with MSFT.
Per Slack thread: https://opi-project.slack.com/archives/C033E418VCK/p1649275592758869. Other opi projects can refer to this.

### CODE REVIEW

File: doc-github-rules.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+### Create a Pull Request
+
+You can make any changes to a fork, including making branches and opening pull requests. 
+If you want to contribute to the original repository, you can submit a pull request.
+
+For more information, see [Creating a pull request from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+
+If you use the Github desktop app, see [Creating an issue or pull request](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request). 
+
+## Create a Pull Request from a branch in the original repository
```
Comment (mestery) [inline doc-github-rules.md]: I would prefer we remove this as I would rather we only do PRs from forks and not from the original repository.

File: doc-github-rules.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+For more information, see [Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). 
+
+If you use the Github desktop app, see [Creating an issue or pull request](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request). 
+
+## PR Review Phase
+Pull requests generally are open to community review and get accepted as-is or with requested changes. The more impactful the changes, the more review activity is to be expected.
+
+## PR Accept Phase
+Once a PR has received sufficient review and concerns are satisfied, a designated person with sufficient [permissions](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role) can accept the Pull Request.
```
Comment (mestery) [inline doc-github-rules.md]: Maybe link to the [maintainers](https://github.com/opiproject/opi/blob/main/MAINTAINERS.md) file here?

File: doc-github-rules.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+If you use the Github desktop app, see [Creating an issue or pull request](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request). 
+
+## PR Review Phase
+Pull requests generally are open to community review and get accepted as-is or with requested changes. The more impactful the changes, the more review activity is to be expected.
+
+## PR Accept Phase
+Once a PR has received sufficient review and concerns are satisfied, a designated person with sufficient [permissions](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization#permissions-for-each-role) can accept the Pull Request. 
+
+## Directly Editing Artifacts
+Contributors with write access permission or higher have rights to directly edit documents and push changes. This practice should be used very sparingly, and only in the draft stages of documents, or to make trivial changes in documents already in use.
```
Comment (mestery) [inline doc-github-rules.md:66]: Excellent wording, I agree with this!

**Review by mestery — CHANGES_REQUESTED** [review]
Good start, some suggestions included in my review.

File: doc-github-rules.md (hunk 2 above)
Comment (chrispsommers) [reply doc-github-rules.md]: Done, thanks for suggestion.

File: doc-github-rules.md (hunk 1 above)
Comment (chrispsommers) [reply doc-github-rules.md]: Done. I agree, that had been a concession to the original project, which uses local branches more than most Community projects.

**Comment by chrispsommers** [comment]
@mestery I addressed all your concerns and resolved all the conversations. Yet for some reason, the PR shows requested changes still need to be addressed.

**Review by mestery — APPROVED** [review]
Excellent work @chrispsommers, thank you!

**Review by glimchb — APPROVED** [review]
lgtm

**Review by pdp2shirts — APPROVED** [review]

**Pull request merged**